### PR TITLE
redirect Note-Actions instead of forwarding

### PR DESCRIPTION
**Pull request by tinect**
### 1. Why is this change necessary?
Looks better :-)

### 2. What does this change do, exactly?
change forward to redirect

### 3. Describe each step to reproduce the issue or behaviour.
- add product to note
- switch to note
- remove product from note
- see address in browser

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: engine/Shopware/Controllers/Frontend/Note.php [hunk 1]
```diff
@@ -62,7 +63,7 @@ public function addAction()
             $this->View()->sArticleName = Shopware()->Modules()->Articles()->sGetArticleNameByOrderNumber($orderNumber);
         }
 
-        $this->forward('index');
+        $this->redirect(['action' => 'index']);
```
Comment (uehler) [inline engine/Shopware/Controllers/Frontend/Note.php:66]: you will loose the view variable with the redirect, won't you?

Comment (shyim) [reply engine/Shopware/Controllers/Frontend/Note.php:66]: You lose in forward and redirect the view variables. I guess the important change is here, that the GET/POST parameters are away.

**Comment by shyim** [comment]
Please change the target to 5.6. It's a BC break

**Comment by shyim** [comment]
Hey @tinect,
your change has been merged in 5.6. Thanks for your Contribution! :blue_heart:

**Pull request closed without merging**